visualization/FairnessDashboard/.eslintrc.js

```diff
@@ -0,0 +1,32 @@
+module.exports = {
+    parser: '@typescript-eslint/parser',
+    parserOptions: {
+        ecmaVersion: 2020,
+        sourceType: 'module',
+        ecmaFeatures: {
+            jsx: true, // Allow parsing of JSX
+        },
+    },
+    settings: {
+        react: {
+            version: 'detect', // Tells eslint-plugin-react to automatically detect the version of React to use
+        },
+    },
+    extends: [
+        'plugin:react/recommended',
+        'plugin:@typescript-eslint/recommended', // Recommended rules from the @typescript-eslint/eslint-plugin
+        'prettier/@typescript-eslint',
+        'plugin:prettier/recommended',
+    ],
+    rules: {
+        '@typescript-eslint/no-this-alias': 'off',
+        'react/display-name': 'off',
+        '@typescript-eslint/no-empty-function': 'off',
+        '@typescript-eslint/no-empty-interface': 'off',
+        '@typescript-eslint/no-var-requires': 'off',
+        'react/prop-types': 'off',
+        'react/jsx-key': 'off',
+        'react/no-unescaped-entities': 'off',
+        'react/no-direct-mutation-state': 'off',
```

Comment:
I'm guessing these are all teh places that raised errors when run, and so for now are turned off? most seem fine, we can try to enable some in a future pr

Reply:
yes, exactly